recipes/freebayes/1.2.0/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -e
+
+export CXXFLAGS="-I$PREFIX/include"
+export LDFLAGS="-L$PREFIX/lib"
+export CPPFLAGS="-I$PREFIX/include"
+
+make -j 1
+cp bin/freebayes bin/bamleftalign $PREFIX/
```

Comment:
The binaries should go in `$PREFIX/bin` (newer versions of `conda-build` will complain `'freebayes' not in tarball`).
If there's no `install` target then `$PREFIX/bin` may not exist, so it's safest to `mkdir -p "$PREFIX/bin"` first.